web/src/server_events_dispatch.js

```diff
@@ -210,9 +210,15 @@ export function dispatch_normal_event(event) {
                 allow_message_editing: noop,
                 avatar_changes_disabled: settings_account.update_avatar_change_display,
                 bot_creation_policy: settings_bots.update_bot_permissions_ui,
+                can_add_custom_emoji_group: noop,
+                can_create_groups: noop,
+                can_create_private_channel_group: noop,
+                can_create_public_channel_group: noop,
```

Comment:
We can replace `noop` in some of the cases here with the function that is being called for that setting. Will do this as a follow-up.

Reply:
Sounds good!

Reply:
Could also be reasonable to construct this data structure in part using `realm.server_supported_permission_settings`, to remove the need to update this file just to live-update these settings properly.

Reply:
Yes, we can do that. Will look into it.